app/models/flag.rb

```diff
@@ -10,6 +10,7 @@ class Flag < ActiveRecord::Base
   MAX_DESCRIPTION_LENGTH = 1000
   scope :enabled, -> { where(enabled: true) }
   scope :system, -> { where("id < 1000") }
+  scope :custom, -> { where("id >= 1000") }
```

Comment:
Just a thought but I wonder if this can just be `not.system` so we don't have to duplicate the `1000` magic number across two scopes. Anyway, not a big problem just a thought I had.